pyeudiw/satosa/backends/openid4vp/endpoints/request_endpoint.py

```diff
@@ -48,6 +54,17 @@ def __init__(
 
         self.metadata_jwks_by_kids = {i["kid"]: i for i in self.config["metadata_jwks"]}
         self.trust_evaluator = trust_evaluator
+        self._credential_supported_formats = [
+            f["format"] for f in config["credential_presentation_handlers"]["formats"]
+        ]
+
+        client_id_schemes = []
+        for _, value in config["trust"].items():
+            client_id = value.get("config", {}).get("client_id")
+            if client_id:
+                prefix = client_id.split(":", 1)[0]
```

Comment:
what about the presence of //, like `https://`

by splitting only `:`, do we assume that the client_id would be therefore set with `//example.org`?